index.ts

```diff
@@ -144,6 +144,8 @@ collect.set('isNotifications', [
 
 export const isOrganizationProfile = (): boolean => exists('meta[name="hovercard-subject-tag"][content^="organization"]');
 
+export const isOrganizationRepo = (): boolean => document.querySelector<HTMLFormElement>('[data-owner-scoped-search-url]')!.dataset.ownerScopedSearchUrl!.startsWith('/org');
```

Comment:
Detections must run without errors
```suggestion
export const isOrganizationRepo = (): boolean => Boolean(document.querySelector<HTMLElement>('[data-owner-scoped-search-url]')?.dataset.ownerScopedSearchUrl!.startsWith('/org'));
```